packages/react-scripts/template/README.md

```diff
@@ -1452,6 +1452,7 @@ If this doesn’t happen, try one of the following workarounds:
 * Some editors like Vim and IntelliJ have a “safe write” feature that currently breaks the watcher. You will need to disable it. Follow the instructions in [“Working with editors supporting safe write”](https://webpack.github.io/docs/webpack-dev-server.html#working-with-editors-ides-supporting-safe-write).
 * If your project path contains parentheses, try moving the project to a path without them. This is caused by a [Webpack watcher bug](https://github.com/webpack/watchpack/issues/42).
 * On Linux and macOS, you might need to [tweak system settings](https://webpack.github.io/docs/troubleshooting.html#not-enough-watchers) to allow more watchers.
+* Inside virtual machines such as (a Vagrant provisioned) VirtualBox, [use `CHOKIDAR_USEPOLLING=true npm start`](https://github.com/facebookincubator/create-react-app/issues/1049#issuecomment-261731734).
```

Comment:
Can this problem occur on Windows? If it can, we should recommend running `npm i --save-dev cross-env` and then replacing `npm start` with `cross-env CHOKIDAR_USEPOLLING=true npm start`.